apps/cli/templates/vercel/remix.config.js

```diff
@@ -0,0 +1,31 @@
+/** @type {import('@remix-run/dev').AppConfig} */
```

Comment:
same here

Reply:
btw why do we 2 identical configs?

Reply:
Not needed, we should pick same files from the `defaults` folder 👍

Reply:
`vercel` seems to need their own config for server.

```js
server: process.env.NODE_ENV === "development" ? undefined : "./server.ts",
```

Not sure, if patching the default with this line is worth the time spent in dev setup at the moment. So, just left the duplicate file. Until there can be a scalable setup for this thing. To extend the default with specific template changes.